README.md

```diff
@@ -1,2 +1,23 @@
 # HW01
-Edit the readme
+Nabilah Sammudin
+7/6/2020
+
+## Brief Biography
+
+My name is **Nabilah** and I am from **Malaysia**. I graduated from **Purdue University** back in 2017 and I am currently working as a research technician in the same lab I did my research as an undergraduate. 
+
+I am interested in studying *Learning and Memory* at the *cell, molecular level* as well as potentially using *computational techniques* in my research. Although I have very little background in coding, I believe that it is an invaluable skill in doing my PhD and I am excited to learn more through this class!
```

Comment:
It's great that you want to start learning about coding / computational techniques.  When I first started learning how to code it was definitely frustrating (and still is at times, especially when learning a new language) but it is very useful.  I have worked in MATLAB a handful of times, but I have no experience using R so looks like we will be learning together through this course!